irmark1/parts/controller.py

```diff
@@ -890,8 +890,9 @@ def chaos_monkey_off(self):
         self.chaos_monkey_steering = None
 
 
-    def run_threaded(self, img_arr=None):
+    def run_threaded(self, img_arr=None, rear_img_arr=None):
```

Comment:
Is the rear_img_arr used?

Reply:
It isn't used yet. The reason I put it here is that I thought in the future we would need rear_img in autonomous driving. Since we have agreed that we don't need it, you can remove this part.